addons/gut/signal_watcher.gd

```diff
@@ -63,7 +63,7 @@ func _add_watched_signal(obj, name):
 		_watched_signals[obj] = {name:[]}
 	else:
 		_watched_signals[obj][name] = []
-	obj.connect(name, self, '_on_watched_signal', [obj, name])
+	obj.connect(name,Callable(self,'_on_watched_signal'),[obj,name])
```

Comment:
I'm not sure if this will work or not, but it is closer than it was.  This could use a `# !4.0 comment`.

Reply:
I think we could even change it simply to: _on_watched_signal instead of this Callable wrapper stuff